src/components/Select/Select.stories.mdx

```diff
@@ -0,0 +1,214 @@
+import { action } from "@storybook/addon-actions";
+import { getCategory } from "../../utils/componentCategories";
+
+<Meta
+  title={getCategory("Select")}
+  component={Select}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=11895%3A549",
+    },
+  }}
+  argTypes={{
+    blockName: { table: { disable: true } },
+    className: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+    attributes: { table: { disable: true } },
+  }}
+/>
+
+# Select
+
+<Description of={Select} />
+
+The `Select` component renders a `select` element along with its `option` children. For optimal accessibility, the `labelText` property is a required prop, regardless of the label visibility. Additionally, while the `id` prop is optional, a unique `id` attribute is necessary for accessibility. If the prop is left blank, a value will be generated for you.
+
+<Preview withToolbar>
+  <Story
+    name="Select"
+    args={{
+      id: "mySelect",
+      labelText: "What is your favorite color?",
+      showLabel: true,
+      helperText: "Choose wisely.",
+      errorText: "Waaaaaaaa!",
+      errored: false,
+      required: true,
+      disabled: false,
+      selectedOption: "Option Two",
+      attributes: {
+        onFocus: action("focus"),
+        onBlur: action("blur"),
+        onChange: action("onChange"),
+      },
+    }}
+  >
+    {args => (
+      <Select {...args}>
+        <option aria-selected={true}></option>
+        <option aria-selected={false}>Red</option>
+        <option aria-selected={false}>Green</option>
+        <option aria-selected={false}>Blue</option>
+        <option aria-selected={false}>Black</option>
+        <option aria-selected={false}>White</option>
+      </Select>
+    )}
+  </Story>
+</Preview>
+
+<ArgsTable story="Select" />
+
+## Labelling Variations
+
+A Select can be rendered with or without visible labels.
+
+<Preview withToolbar>
+  <Story
+    name="Labelling Variations"
+    args={{
+      labelText: "What is your favorite color?",
+      showLabel: false,
+    }}
+  >
+    {args => (
+      <Select {...args}>
+        <option aria-selected={true}></option>
+        <option aria-selected={false}>Red</option>
+        <option aria-selected={false}>Green</option>
+        <option aria-selected={false}>Blue</option>
+        <option aria-selected={false}>Black</option>
+        <option aria-selected={false}>White</option>
+      </Select>
+    )}
+  </Story>
+</Preview>
+
+<Canvas>
+  <Select labelText="What is your favorite color?" showLabel={true}>
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
```

Comment:
Can you add a preface here saying that this is how the `helperText` prop gets rendered?

Reply:
Sorry, I meant this for the next example on line 127 when `showLabel` is set to true.

Reply:
I am not sure we need to explain the details behind the variations.  Adjusting the props in the ArgsTable should suffice.